dev/Dockerfile

```diff
@@ -6,10 +6,10 @@ LABEL maintainer="pangxiaoxi@gmail.com, cndream@gmail.com"
 LABEL description="dv-ingestor dev docker image"
 
 ARG DEV_HOME
+ENV APP_HOME $DEV_HOME/app
```

Comment:
Get into app folder on start so that this will help avoiding accidentally install package in repo root folder